### PR TITLE
Updates the "inLastN" property of a VideoChannel when changes to the …

### PR DESCRIPTION
**Pull request by bgrozev**
…"lastN" set occur.

Torture passed, manually tested with the scenario described in #132.

### CODE REVIEW

**Comment by davidertel** [comment]
I will do some testing on this tomorrow, thanks for the fix!

File: src/main/java/org/jitsi/videobridge/LastNController.java [hunk 1]
```diff
@@ -544,6 +551,91 @@ else if (newForwardedEndpoints.size() < currentLastN)
...
+        return getVideoChannels(endpoint);
+    }
+
+    /**
+     * Gets the list of {@link VideoChannel}s belonging to an endpoint.
+     *
+     * @param endpoint the endpoint.
+     * @return the {@link VideoChannel}s of the endpoint.
+     */
+    private List<VideoChannel> getVideoChannels(Endpoint endpoint)
```
Comment (gpolitis) [inline src/main/java/org/jitsi/videobridge/LastNController.java]: Do we really need this since we have `endpoint.getChannels(MediaType.VIDEO)`?

**Comment by gpolitis** [comment]
LGTM. @davidertel do you still want to do some testing before I merge this?

**Comment by lsevans** [comment]
I just ran some tests and confirmed this fixes the issue I was seeing. Thanks!

**Comment by davidertel** [comment]
George, I did some testing and it looks good to me. Thank you again

**Comment by gpolitis** [comment]
@lsevans, @davidertel thank you!

**Pull request merged**